scripts/jobs/cron_tasks.inc.dns.10.bind.php

```diff
@@ -365,6 +365,17 @@ protected function generateZone($domain, $froxlorhost = false) {
 				$zonefile.= $record . "\tIN\t" . $ip_a_record . "\n";
 			}
 		}
+		
+		// Append custom DNS-Records
+		$domain_stmt = Database::prepare("
+			SELECT `dnsrecords` FROM `" . TABLE_PANEL_DOMAINS . "`
+			WHERE `id` = :domainid
+		");
+		Database::pexecute($domain_stmt, array('domainid' => $domain['id']));
+						
+		while ($dnsrecords = $domain_stmt->fetch(PDO::FETCH_ASSOC)) {
```

Comment:
Why re-read the dnsrecords field, superfluous query with even more superfluous while-loop? Just add the dnsrecords field to the query in line 80 of this file and use 
```php
$zonefile .= $domain['dnsrecords'];
```